app/classifier/tasks/combo/index.cjsx

```diff
@@ -71,13 +71,28 @@ ComboTask = React.createClass
           allComboAnnotations.push annotation.value...
 
       <g className="combo-task-persist-inside-subject-container">
-        {Object.keys(props.taskTypes).map (taskType) ->
+        {Object.keys(props.taskTypes).map (taskType, idx) ->
           unless taskType is 'combo'
             TaskComponent = props.taskTypes[taskType]
             if TaskComponent.PersistInsideSubject?
               fauxClassification =
                 annotations: allComboAnnotations
-              <TaskComponent.PersistInsideSubject key={taskType} {...props} classification={fauxClassification} />}
+                update: () => props.classification.update(arguments)
```

Comment:
I tried that at first and got `Uncaught TypeError: Array.prototype.indexOf called on null or undefined` errors when a drawn point is moved/updated (I think this error has to do with the proper `this` being passed into `update`).